blocks/product-details/product-details.js

```diff
@@ -28,10 +28,23 @@ import { IMAGES_SIZES } from '../../scripts/initializers/pdp.js';
 import '../../scripts/initializers/cart.js';
 import { rootLink } from '../../scripts/scripts.js';
 
+/**
+ * Checks if the current page is for updating a product in the cart
+ * @returns {Object|null} The update info object or null
+ */
+function getUpdateCartInfo() {
+  const itemUid = new URLSearchParams(window.location.search).get('itemUid');
+  return itemUid ? {
+    isUpdating: true,
+    itemUid,
+  } : null;
+}
+
```

Comment:
With the redirected cart page URL provided in PR #419, I believe we could remove this part. 
```suggestion
```